src/PgRvn.Server/Messages/MessageReader.cs

```diff
@@ -360,9 +369,23 @@ private async Task<byte> ReadByteAsync(PipeReader reader, CancellationToken toke
             return ReadByte(read.Buffer, reader);
         }
 
+        private async Task SkipBytesAsync(PipeReader reader, CancellationToken token, int length)
+        {
+            var read = await ReadMinimumOf(reader, length, token);
+            SkipBytes(read.Buffer, reader, length);
+        }
+
+        private void SkipBytes(ReadOnlySequence<byte> readBuffer, PipeReader reader, int length)
+        {
+            var sequence = readBuffer.Slice(0, length);
```

Comment:
Don't think this will work. If the size is 32 MB, for example, you'll error here.

Reply:
I tested this with Npgsql and a `bytea` parameter of 52 MB and it seems to work